core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -358,15 +377,16 @@ public static boolean getBoolean(String key) {
    * @return the bytes of the value for the given key
    */
   public static long getBytes(String key) {
-    if (PROPERTIES.containsKey(key)) {
-      String rawValue = get(key);
-      try {
-        return FormatUtils.parseSpaceSize(rawValue);
-      } catch (Exception ex) {
-        throw new RuntimeException(ExceptionMessage.KEY_NOT_BYTES.getMessage(key));
-      }
+    String rawValue = get(key);
```

Comment:
It seems possible  that `lookup(rawValue) `returns a `null` even though it passes the check inside `get`.